### DISCUSSION TITLE
不会使用

### DISCUSSION THREAD

**Discussion by Leq27** [Mzying2001/CefFlashBrowser#41] [General]
下载完怎么使用啊有没有大佬教学一下

**Comment by Mzying2001** [comment]
1、解压缩

![1](https://github.com/Mzying2001/CefFlashBrowser/assets/41951400/8f6ae27f-a56f-4530-a886-0bf27bff2d79)

2、进入解压缩出来的文件夹，双击 `CefFlashBrowser.exe` 即可打开程序

![2](https://github.com/Mzying2001/CefFlashBrowser/assets/41951400/b1f45e43-871a-4c69-9003-858d23aaf6ca)

**Reply by angelyxh** [reply]
大佬我这咋跟你的不一样啊，没有你那个带exe的压缩包啊
![image](https://github.com/user-attachments/assets/5978da70-3b63-4407-809c-9ba4e08d6bc0)
![image](https://github.com/user-attachments/assets/117cfe25-f514-4b18-922f-71adfccb1eb0)

**Reply by Mzying2001** [reply]
你下载错了，你下的那个是代码，程序的下载方式如下：
![1](https://github.com/user-attachments/assets/3dd2166a-50e1-4a47-a496-252b43bcfa46)
![2](https://github.com/user-attachments/assets/ceeb67a2-3027-44d3-b7fb-35e3f54c4752)

**Reply by angelyxh** [reply]
感谢大佬

**Comment by x88520** [comment]
下载后打不开是怎么回事？
![Uploading image.png…]()

**Comment by x88520** [comment]
System.IO.FileNotFoundException: 未能加载文件或程序集“CefSharp.Core, Version=84.4.10.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138”或它的某一个依赖项。找不到指定的模块。
文件名:“CefSharp.Core, Version=84.4.10.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138” ---> System.IO.FileNotFoundException: 未能加载文件或程序集“CefSharp.Core.dll”或它的某一个依赖项。找不到指定的模块。
文件名:“CefSharp.Core.dll”
   在 System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   在 System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   在 System.Reflection.RuntimeAssembly.InternalLoadFrom(String assemblyFile, Evidence securityEvidence, Byte[] hashValue, AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean suppressSecurityChecks, StackCrawlMark& stackMark)
   在 System.Reflection.Assembly.LoadFrom(String assemblyFile)
   在 CefFlashBrowser.App.<>c.<.ctor>b__1_0(Object s, ResolveEventArgs e)
   在 System.AppDomain.OnAssemblyResolveEvent(RuntimeAssembly assembly, String assemblyFullName)


   在 CefFlashBrowser.App.InitCefFlash()
   在 CefFlashBrowser.App.OnStartup(StartupEventArgs e)

**Reply by x88520** [reply]
显示这样，我试过几个版本都是这样

**Reply by Mzying2001** [reply]
按照这个试试看看行不行：https://github.com/Mzying2001/CefFlashBrowser/issues/38#issuecomment-2253726773

**Reply by x88520** [reply]
好的，谢谢
